src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -627,6 +628,19 @@ internal void Save(string filePath, bool isBackup = false, EngineController engi
                 {
                     // For intentional SaveAs either through UI or API calls, replace workspace elements' Guids and workspace Id
                     jo["Uuid"] = Guid.NewGuid().ToString();
+                    if (jo["Bindings"] != JToken.Parse("[]"))
+                    {
+                        jo["Bindings"] = JToken.Parse("[]");
+                        var result = DynamoMessageBox.Show( Wpf.Properties.Resources.ElementBindingWarningMessage,
+                            Wpf.Properties.Resources.ElementBindingWarningTitle, MessageBoxButton.OKCancel,
```

Comment:
hmm, what about using the !` notifications menu bar, it will also log to the console if you use the LogNotification method so we can note it in debug logs, but it won't interrupt every save as operation as almost all Revit graphs will have element binding.
<img width="319" alt="Screenshot 2023-09-08 at 3 27 45 PM" src="https://github.com/DynamoDS/Dynamo/assets/508936/c24a3d52-4eac-4865-87b4-0a5f4374148e">

Reply:
I'm not sure I agree, I feel we should have the dialog indicating to the user that they're about to lose their element binding data (which is a pretty major regression), at least until we restore legacy save as.